kshebeko/lesson4/src/com/company/kshebeko_class/Bouquet.java

```diff
@@ -0,0 +1,93 @@
+package com.company.kshebeko_class;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Bouquet {
+    public Accessories accessories;
+    private int price;
+
+    public Bouquet() {
+    }
+
+    private ArrayList<Flower> flowerList = new ArrayList();
+
+    public void setFlowerList(ArrayList<Flower> list) {
+        this.flowerList = list;
+        countPriceBouquet();
+    }
+
+    public ArrayList<Flower> flowers = new ArrayList<>();
+
+    public ArrayList getFlowers() { return flowers;}
+
+    @Override
+    public String toString() {
```

Comment:
toString and printFlowers methods return same info, so it is kind of duplicated code